hack/dockerized

```diff
@@ -46,7 +46,7 @@ $KUBEVIRT_CRI run ${CONTAINER_ENV} -v "${BUILDER}:/root:rw,z" --security-opt "la
 mkdir -p ${OUT_DIR}
 
 # Start an rsyncd instance and make sure it gets stopped after the script exits
-RSYNC_CID=$($KUBEVIRT_CRI run ${CONTAINER_ENV} -d -v "${BUILDER}:/root:rw,z" --security-opt "label=disable" --expose 873 -P ${KUBEVIRT_BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
+RSYNC_CID=$($KUBEVIRT_CRI run ${CONTAINER_ENV} -d -v "${BUILDER}:/root:rw,z" --restart unless-stopped --security-opt "label=disable" --expose 873 -P ${KUBEVIRT_BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
```

Comment:
I think this would still be ok as there is a force remove in the finish function. This force remove will clean up the rsync container. 

https://github.com/kubevirt/kubevirt/blob/30d8ed316ca093ee83fc632449f058b2f690696b/hack/dockerized#L51-L53

I may update this to `--restart always` as this is more indicative of the behaviour under podman?